client/clip_client/client.py

```diff
@@ -147,44 +162,34 @@ def _gather_result(self, response, results: 'DocumentArray'):
             ),
         )
 
-    @staticmethod
-    def _unboxed_result(results: 'DocumentArray'):
-        if results.embeddings is None:
-            raise ValueError(
-                'Empty embedding returned from the server. '
-                'This often due to a mis-config of the server, '
-                'restarting the server or changing the serving port number often solves the problem'
-            )
-        return (
-            results.embeddings if ('__created_by_CAS__' in results[0].tags) else results
-        )
-
-    def _iter_doc(self, content) -> Generator['Document', None, None]:
+    def _iter_doc(
+        self, content, results: 'DocumentArray'
+    ) -> Generator['Document', None, None]:
         from rich import filesize
         from docarray import Document
 
         if hasattr(self, '_pbar'):
             self._pbar.start_task(self._s_task)
 
-        for c in content:
+        for i, c in enumerate(content):
```

Comment:
where is this `i` used?

Reply:
it is used to mark the order of input, and it is now deprecated. Will remove it